### PR TITLE
Generate interacts and prepare for publish

### LINKED ISSUE #279: Generate interacts and prepare publish npm

### PR DESCRIPTION
**Pull request by abhayks1**
PR covers:
- Generation of interacts with new contracts
- Implementation of `npm run prepare:publish`

`npm run postinstall` is not needed. After the package is published, the command runs everytime when someone does `npm i`. 
We should do `npm run generate:interacts` for generating interacts.

Fixes #279

### CODE REVIEW

File: tools/build_package.js [hunk 1]
```diff
@@ -34,9 +34,18 @@ const contractNames = [
   'Axiom',
   'Committee',
   'Consensus',
+  'ConsensusCogateway',
+  'ConsensusGateway',
   'Core',
+  'Coreputation',
+  'ERC20Cogateway',
+  'ERC20Gateway',
```
Comment (0xsarvesh) [inline tools/build_package.js:43]: Add Gen0ERC20Cogateway

File: tools/build_package.js [hunk 2]
```diff
@@ -34,9 +34,18 @@ const contractNames = [
...
   'ERC20I',
   'Reputation',
+  'SelfProtocore',
+  'StateRootProvider',
+  'UtBase',
+  'UtilityToken',
```
Comment (0xsarvesh) [inline tools/build_package.js:52]: Should we add MessageBus

**Review by 0xsarvesh — CHANGES_REQUESTED** [review]
Overall looks good 👍 
Minor comment

File: tools/build_package.js (hunk 1 above)
Comment (abhayks1) [reply tools/build_package.js:43]: Done!

File: tools/build_package.js (hunk 2 above)
Comment (abhayks1) [reply tools/build_package.js:52]: Done!

**Review by 0xsarvesh — APPROVED** [review]
LGTM 🚀

**Pull request merged**